hashicorp/nomad
# website: update dependencies

### Description

Dev Portal was recently updated to use [Next V14](https://github.com/hashicorp/dev-portal/blob/main/package.json#L177). The package.json in the Nomad website directory now has updated versions of next, node, npm, husky and other HashiCorp platform packages. This is to prevent any errors during the deploy preview process and local development, which has been an issue for other product teams when developing locally & deploying changes to Dev Portal.

### Links

- [Asana task](https://app.asana.com/0/0/1206444672170572/f)


### Testing & Reproduction steps

- cd into website directory
- run `nvm use` -> should use node v18.17+ and npm v.9.6.7+
- run `npm install` -> should execute without errors
- run `npm run start` -> should execute without errors
- visit localhost:3000 -> should work as expected without errors
